cg/models/delivery/delivery.py

```diff
@@ -0,0 +1,12 @@
+"""Delivery API related models."""
+
+from pathlib import Path
+
+from pydantic import BaseModel, FilePath
+
+
+class DeliveryFile(BaseModel):
+    """Model representing a file to be delivered."""
+
+    source_path: FilePath
```

Comment:
Is there a difference between FilePath and Path?

Reply:
Yes! A `FilePath` must point to an existing file